src/main/java/net/snowflake/ingest/connection/SecurityManager.java

```diff
@@ -120,6 +120,33 @@
     this(accountname, username, keyPair, RENEWAL_INTERVAL, TimeUnit.MINUTES);
   }
 
+  /**
+   * Trims an account name if it contains a <b>"."</b>
+   *
+   * <p>Snowflake's python connector trims an accountname if it contains a "." </p>
+   *
+   * @param accountName given accountName in SimpleIngestManager's constructor
+   * @return initial part of account name if originally it contained a ".", otherwise return the same accountName.
+   * @see <a href="https://github.com/snowflakedb/snowflake-connector-python/blob/master/src/snowflake/connector/util_text.py#L227">Python Connector</a>
+   */
+  private String parseAccount(final String accountName)
+  {
+    String parseAccount = null;
+    if (accountName.contains("."))
+    {
+      final String[] accountParts = accountName.split("[.]");
+      if (accountParts.length > 1)
```

Comment:
NIT: Is there a valid scenario where this is false? Can we remove it?

Reply:
Very rare. In cases like they pass an accountname which ends in a dot may be?